### PR TITLE
try ubuntu-24.04

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         config:
           - name: Linux GCC
-            os: ubuntu-latest
+            os: ubuntu-24.04
```
Comment (fabiangreffrath) [inline .github/workflows/main.yml:25]: Also for the cppcheck task, please.

Comment (rfomin) [reply .github/workflows/main.yml:25]: Looks like very few VMs are available, we'll have to wait.

**Comment by rfomin** [comment]
It looks like we will have to wait until August.

**Comment by fabiangreffrath** [comment]
Yes, we can wait until 24.04 becomes "ubuntu-latest". Until then, I'm compiling the code nearly daily with Debian testing, so I don't expect any surprising results by switching Ubuntu releases anyway.

**Pull request closed without merging**